pkg/build/node/node.go

```diff
@@ -415,6 +415,29 @@ func (c *BuildContext) prePullImages(dir, containerID string) error {
 		log.Errorf("Image build Failed! %v", err)
 		return err
 	}
+
+	// systemd-in-a-container should have read only /sys
+	// https://www.freedesktop.org/wiki/Software/systemd/ContainerInterface/
+	// however, we need other things from `docker run --privileged` ...
+	// and this flag also happens to make /sys rw, amongst other things
+	// Need --privileged flag in docker exec
+	if err = execInBuild("mount", "-o", "remount,ro", "/sys"); err != nil {
```

Comment:
why is this happening at build time?? mounts are created when the container is created.